public_html/wp-content/plugins/wc-post-types/inc/favorite-schedule-shortcode.php

```diff
@@ -181,6 +183,12 @@ function preprocess_schedule_attributes( $attr ) {
 		$attr['session_link'] = 'permalink';
 	}
 
+	// Sites without the `content_blocks` skip flag use blocks, so we need to prevent these from using `anchor`.
```

Comment:
`SHOULD` - I think it'd be good to explain the reasoning here more. A new developer reading this -- or ourselves in 6 months -- probably wouldn't get enough information from the comment in order to understand the problem that's being solved.

It might be best to add all that explanation in the comment above, though, and make this comment just say something like "see above for details".